test/config_test.py

```diff
@@ -170,3 +170,38 @@ def test_signing(self):
             set(re.search(r'headers="(.+?)"', post_sig).group(1).split(" ")),
             {"(request-target)", "date", "host", "content-length", "digest"},
         )
+
+    def test_bearer_token(self):
+        """Verify that the authorization header is set when a bearer token is provided"""
+
+        bearer_token = "Bearer myBearerToken"
+
+        with tempfile.NamedTemporaryFile() as config_file, tempfile.NamedTemporaryFile() as key_file:
+            with open(config_file.name, "w") as f:
+                json.dump(
+                    {
+                        "email": "somebody@transcriptic.com",
+                        "token": bearer_token,
```

Comment:
Design decision here. The user token field here we've used historically is fundamentally quite different from bearer tokens (which are under a more traditional Auth framework). 

I think its fine for now, but something to consider with overloading this field is support for other authentication types in the future. For example, Basic, OAuth, etc. all use the "token" nomenclature as well, but have different schema.

An alternative would be to have a more explicit `authentication` field, with a type and relevant fields.

Reply:
Yeah, that was my initial approach, but I chose to be kind of surgical with the change to enable backwards compatibility for existing programs. I.e. let Igor pass them a bearer token (instead of a out x-user-token), which they'll use to create a connection.